core/services/relay/evm/functions.go

```diff
@@ -23,8 +23,8 @@ func (p *functionsProvider) ContractTransmitter() types.ContractTransmitter {
 	return p.contractTransmitter
 }
 
-func NewFunctionsProvider(chainSet evm.ChainSet, rargs relaytypes.RelayArgs, pargs relaytypes.PluginArgs, lggr logger.Logger, ethKeystore keystore.Eth, eventBroadcaster pg.EventBroadcaster) (relaytypes.Plugin, error) {
```

Comment:
Functions does not appear use the recently-added eventBroadcaster.  Is this something you plan to use in the future? Else I will keep it removed.